Spec/RealtimeClientConnection.swift

```diff
@@ -2390,13 +2395,14 @@ class RealtimeClientConnection: QuickSpec {
                                 done()
                             }
                         }
-                        expect(channel.state).to(equal(ARTRealtimeChannelState.detached))
+                        expect(client.msgSerial).to(equal(0))
+                        expect(channel.state).to(equal(ARTRealtimeChannelState.attaching))
```

Comment:
@tcard I changed the test regarding the [last comment](https://github.com/ably/ably-ios/issues/612#issuecomment-306760888) of @SimonWoolf. /cc @mattheworiordan